ai/kai-assistant/index.md

```diff
@@ -1,77 +1,84 @@
+**Perfect for:**
+- ✅ **Data engineers** building and maintaining data pipelines
+- ✅ **Analytics engineers** creating SQL and Python transformations
+- ✅ **Project managers** monitoring jobs and debugging issues
+- ✅ **New team members** learning Keboola best practices
+- ✅ **Anyone working with data** who wants to be more productive
 
-**Integration Setup**
-- Configure data extractors (Shopify, Google Sheets, APIs, CSV)
-- Set up writers and data destinations
-- Create custom Python components for specialized integrations
-- Manage API tokens and authentication
-- Add packages to transformation environments
+## What You Can Do With KAI
 
-**Coding & Development**
-- Write SQL transformations
-- Create Python transformations
-- Convert between Python and SQL
-- Optimize queries and transformation logic
-- Build custom data processing components
+**Analyze & Explore**  
+Execute SQL queries, explore schemas, calculate metrics, analyze trends
+→ *"Show me total revenue by month from the transactions table"*
 
-**Troubleshooting**
-- Debug job failures with automated diagnosis
-- Resolve configuration errors
-- Fix transformation bugs
-- Investigate data quality issues
-- Handle API token and authentication issues
+**Build & Configure**  
+Set up extractors, writers, transformations, and custom components
+→ *"Create a Snowflake writer for the customer_summary table"*
 
-**Documentation**
-- Generate comprehensive project documentation
-- Update table and column descriptions
-- Document transformations and data flows
-- Create project overviews for new team members
+**Debug & Fix**  
+Diagnose failures, resolve errors, fix bugs, investigate data issues
+→ *"Why did flow 'daily-etl' fail last night?"*
 
-**Data Modeling**
-- Build complex analytical frameworks
-- Create hierarchical data structures
-- Design dimensional models
-- Advanced clustering and taxonomy creation
+**Document & Model**  
+Generate documentation, describe tables, design data models
+→ *"Document what transformation 'revenue-calc' does"*
 
-## How Kai Differs from Other AI Tools
+[See detailed examples for each capability →](use-cases/)
 
-### Context Awareness
-Unlike ChatGPT or Claude, Kai has direct access to your Keboola project. When you ask about an error, it doesn't guess based on syntax—it reads the actual job logs, configurations, and data structures to provide specific, actionable solutions.
+## How KAI Differs from Other AI Tools
 
-### Embedded Experience
-Kai is built into the Keboola platform with no installation required. It's already authenticated and understands your current location within the platform, making it seamless to use.
+| Feature | ChatGPT / Claude | KAI |
+|---------|------------------|-----|
+| **Context** | No access to your project | Full access to configs, schemas, job logs |
+| **Actions** | Can only suggest code | Creates, modifies, and runs components |
+| **Authentication** | Manual setup required | Already authenticated in platform |
```

Comment:
If we are comparing generic AI Assistant with no MCP than authentication is not a valid difference.